src/test/java/seedu/address/logic/parser/EditCommandParserTest.java

```diff
@@ -81,14 +74,6 @@ public void parse_invalidValue_failure() {
 
         // invalid phone followed by valid email
         assertParseFailure(parser, "1" + INVALID_PHONE_DESC + EMAIL_DESC_AMY, Phone.MESSAGE_CONSTRAINTS);
-
-        // valid phone followed by invalid phone. The test case for invalid phone followed by valid phone
-        // is tested at {@code parse_invalidValueFollowedByValidValue_success()}
-        assertParseFailure(parser, "1" + PHONE_DESC_BOB + INVALID_PHONE_DESC, Phone.MESSAGE_CONSTRAINTS);
```

Comment:
I think this should not be deleted. Remove `PHONE_DESC_BOB` and it should still work.